js/components/user/list.vue

```diff
@@ -58,6 +58,13 @@ export default {
                 align: 'center',
                 type: 'metric',
                 width: 95
+            }, {
+                label: this._('Roles'),
+                key: 'roles',
+                sort: 'roles',
```

Comment:
OK, right now this is the case, but more roles are coming very very soon (it's not the place to discuss thoses). So the question is still legit: what happens when a user have 2 roles ?